packit_service/worker/jobs.py

```diff
@@ -311,21 +311,24 @@ def process_jobs(
                     # failed because of missing whitelist approval
                     whitelist = Whitelist()
                     if not whitelist.is_approved(project.namespace):
-                        logger.error(
-                            f"User {project.namespace} is not approved on whitelist!"
-                        )
-                        # TODO also check blacklist,
-                        # but for that we need to know who triggered the action
-
-                        commit_sha = nested_get(event, "pull_request", "head", "sha")
-                        r = BuildStatusReporter(project, commit_sha)
-                        msg = "Account is not whitelisted!"
-                        r.report("failure", msg, url=FAQ_URL)
-
-                        handlers_results[job.job.value] = HandlerResults(
-                            success=False, details={"msg": msg}
-                        )
-                        return handlers_results
+                        if trigger == JobTriggerType.pull_request:
```

Comment:
So, this pretty much means that I'd be able to create a new github project, install packit service, and start committing to fedora dist-git wherever I wanted. Without being approved.

@rpitonak will you fix that in your PR or should it be fixed here?

Reply:
ohh, you are right, the condition can be moved only to `commit_sha` row. If we get rid of commit_sha, then no condition is needed.

Reply:
@TomasTomecek Yes, it is going to check whitelist for both triggers `release` and `pull_request` and then reports status just for the PR.